internal/cli/stop.go

```
@@ -0,0 +1,49 @@
+	"strings"
+
+	"github.com/happyhackingspace/vulnerable-target/pkg/options"
+	"github.com/happyhackingspace/vulnerable-target/pkg/provider/registry"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+var stopCmd = &cobra.Command{
+	Use:   "stop",
+	Short: "Stop vulnerable enviroment by template id and provider",
+	Run: func(cmd *cobra.Command, args []string) {
```

golangci-lint (1.64.8)

[warning] unused-parameter: parameter 'args' seems to be unused, consider removing or renaming it as _

(revive)